application/modules/home/views/side_ci.php

```diff
@@ -0,0 +1 @@
+<h3 class="box-title"><i class="fa fa-building"></i><?= $exhibition['name']?> に加盟している会社情報編集</h3>
```

Comment:
__[code_sniffer]__
* Short PHP opening tag used with echo; expected "<?php echo $exhibition ..." but found "<?= $exhibition ..."
* Any char after the closing PHP tag is prohibited. Please removes newline or spaces after the closing PHP tag.

<!-- Comment by [SideCI](https://sideci.com) -->